introduction.md

```diff
@@ -0,0 +1,6 @@
+My name is Pedro Henrique I'm a developer looking for new knoledges in rails and javascript, but my favourite is the ruby path
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.